log/zap_wrapper.go

```diff
@@ -0,0 +1,41 @@
+package log
+
+import (
+	"go.uber.org/zap/zapcore"
+)
+
+// KitWrapper is a type that implements go-kit/log.Logger interface with zap logger.
+type KitWrapper zapcore.Level
+
+// Log implements go-kit/log.Logger interface.
+func (w KitWrapper) Log(keyvals ...interface{}) error {
+	const msg = ""
+	switch zapcore.Level(w) {
+	default:
+		// for unknown values, fallback to info level.
+		fallthrough
+	case zapcore.InfoLevel:
+		Infow(msg, keyvals...)
+	case zapcore.DebugLevel:
+		Debugw(msg, keyvals...)
+	case zapcore.ErrorLevel:
+		Errorw(msg, keyvals...)
+	case zapcore.PanicLevel:
+		Panicw(msg, keyvals...)
+	case zapcore.FatalLevel:
+		Fatalw(msg, keyvals...)
+	case ZapNopLevel:
+		// do nothing
+	}
+	return nil
+}
+
+// KitLogger returns a go-kit compatible logger.
+func KitLogger(level Level) KitWrapper {
+	return KitWrapper(StringToAtomicLevel(level))
+}
+
+// SimpleLog is a baseplate.go and apache thrift compatible log wrapper.
+func SimpleLog(msg string) {
+	Error(msg)
+}
```

Comment:
This function should be moved to wrapper.go, then it should be made to support levels (similar to the go-kit wrapper):

```go
func ZapWrapper(level zapcore.Level) Wrapper {
	return func(msg) {
		switch zapcore.Level(w) {
		default:
			// for unknown values, fallback to info level.
			fallthrough
		case zapcore.InfoLevel:
			Info(msg)
		case zapcore.DebugLevel:
			Debug(msg)
		case zapcore.ErrorLevel:
			Error(msg)
		case zapcore.PanicLevel:
			Panic(msg)
		case zapcore.FatalLevel:
			Fatal(msg)
		case ZapNopLevel:
			// do nothing
		}
	}
}
```

After you moved this function out, this file can be renamed to kit_wrapper.go as that's what it does.